tensorly/regression/tests/test_cp_plsr.py

```diff
@@ -16,6 +16,10 @@
     tl.get_backend() in ("tensorflow",),
     reason=f"Operation not supported in {tl.get_backend()}",
 )
+skip_if_paddle = pytest.mark.skipif(
+    tl.get_backend() == "paddle",
+    reason=f"Operation not supported in {tl.get_backend()}",
```

Comment:
Since paddle do not support `Tensor_x[bool_mask] = Tensor_y`(assigning tensor to a mask slice tensor), I implemented it with combination of apis below:

https://github.com/tensorly/tensorly/pull/565/commits/67ac03a0b8ff61f8e3dab30409be47cf8b8d183b#diff-f495243586a2d2cfc90c4858babef0d80bb35207b402d9b91d9690cdb012da44R134-R145